base/applications/cmdutils/taskkill/taskkill.c

```diff
@@ -208,6 +210,32 @@ static BOOL get_process_name_from_pid(DWORD pid, WCHAR *buf, DWORD chars)
  *
  * A PID of zero causes taskkill to warn about the inability to terminate
  * system processes. */
+
+
+static DWORD *find_child_pid(DWORD ppid, DWORD *list_count)
+{
+    DWORD* pid_list = HeapAlloc(GetProcessHeap(), 0, 1024 * sizeof(DWORD));
+    DWORD count = 0;
+    HANDLE h = CreateToolhelp32Snapshot(TH32CS_SNAPPROCESS, 0);
+    PROCESSENTRY32 pe = { 0 };
```

Comment:
Please use the Unicode variant of the structure explicitly.

Reply:
Thank you, I fixed